ddl/db_integration_test.go

```diff
@@ -2155,26 +2153,27 @@ func (s *testIntegrationSuite7) TestAddExpressionIndexOnPartition(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
 	tk.MustExec("drop table if exists t;")
-	s.tk.MustExec(`create table t(
+	tk.MustExec(`create table t(
 	a int,
 	b varchar(100),
-	c int,
+	c int)
 	PARTITION BY RANGE ( a ) (
 	PARTITION p0 VALUES LESS THAN (6),
 		PARTITION p1 VALUES LESS THAN (11),
 		PARTITION p2 VALUES LESS THAN (16),
 		PARTITION p3 VALUES LESS THAN (21)
 	);`)
 	tk.MustExec("insert into t values (1, 'test', 2), (12, 'test', 3), (15, 'test', 10), (20, 'test', 20);")
-	tk.MustExec("alter table t add index idx((a+c));")
+	// TODO: Uncomment lines below after fixing #18132.
```

Comment:
You can try L2058, the begining of TestAddExpressionIndex. I think it' will work after that. `config.UpdateGlobal(func(conf *config.Config) { conf.Experimental.AllowsExpressionIndex = true })`

Reply:
Yes, I added that line and the test passed
PTAL https://github.com/pingcap/tidb/pull/18003/commits/32e3a7b9b318375ffe6110df2221e5d2879675df

Reply:
Well done!